docs/user_guide/postprocessing_results.md

```diff
@@ -0,0 +1,76 @@
+# Postprocessing results
+
+When running H2Integrate, results from the simulation and individual technologies are generated automatically.
+Additionally, the raw numerical results are available in the resulting Python object `prob` after a simulation.
+This doc page will walk you through the steps to postprocess the results from a simulation.
+
+```{note}
+Streamlining the postprocessing of results is an ongoing effort in H2Integrate -- please expect this page to be updated as new features are added.
+```
+
+## Automatically generated results
+
+At the conclusion of a simulation, H2Integrate automatically prints a list of all the inputs and outputs for the model to the terminal.
+Here is a snippet of the output from a simulation:
+
+```text
+37 Explicit Output(s) in 'model'
```

Comment:
This can be pinned as a comment for later, but this output feels like we should be utilizing rich or something similar for outputting tabular-ish data in the terminal.

Reply:
That's a great idea! Funny enough, this output is straight from OpenMDAO. Luckily, that's an open-source project where I can suggest they use rich for this. :)

Reply:
It's also a pandas dependency, so it's not entirely adding a new dependency to the stack, just a direct dependency.